src/databricks/labs/ucx/source_code/python_libraries.py

```diff
@@ -37,7 +37,7 @@ def with_next_resolver(self, resolver: BaseLibraryResolver) -> PipResolver:
 
     def resolve_library(self, path_lookup: PathLookup, library: Path) -> MaybeDependency:
         library_path = self._locate_library(path_lookup, library)
-        if library_path is None:  # not installed yet
+        if library_path is None or library_path.suffix == ".whl":  # not installed yet
```

Comment:
Locate dist info has a bug, as it may pick up incorrect wheel to determine dist info. See Wheels class from blueprint dependency.